content/en/tutorials/lotus-miner/cuda.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "CUDA for the Nvidia cards"
+description: "This is a step by step guide on how to configure and use CUDA architecture for Lotus instead of OpenCL for Nvidia cards."
+lead: "This is a step by step guide on how to configure and use CUDA architecture for Lotus instead of OpenCL for Nvidia cards."
+draft: false
+menu:
+    tutorials:
+        parent: "tutorials-providers"
+weight: 130
+toc: true
+---
+
+## Installing latest stable Nvidia Drivers
+
+Note that many Linux distributions provide their own packages of the NVIDIA Linux Graphics Driver in the distribution's native package management format. This may interact better with the rest of your distribution's framework, and you may want to use this rather than NVIDIA's official package.
+
+1. Download the latest stable package for your card using [here.](https://www.nvidia.com/download/index.aspx)
+
+2. Once you have downloaded the driver, change to the directory containing the driver package and install the driver by running, as root, sh ./NVIDIA-Linux-x86_64-<driver version>.run
+Eaxmple:
+
+```shell
+mkdir ~/nvidia-drivers
+cd ~/nvidia-drivers
+wget https://us.download.nvidia.com/XFree86/Linux-x86_64/510.54/NVIDIA-Linux-x86_64-510.54.run
+sudo sh NVIDIA-Linux-x86_64-510.54.run
+```
+
+3. One of the last installation steps will offer to update your X configuration file. Either accept that offer, edit your X configuration file manually so that the NVIDIA X driver will be used, or run nvidia-xconfig
```

Comment:
This is the first time you've mentioned an X configuration file. What is it?

Reply:
X-server config on a linux system. The naming convention is weird but that is exactly what the file is called. I have added the following to hint about x-server.

One of the last installation steps will offer to update your X configuration file(for X-server)